docs/generated/packages/angular/generators/stories.json

```diff
@@ -25,14 +25,6 @@
         "x-priority": "important",
         "default": true
       },
-      "generateCypressSpecs": {
-        "type": "boolean",
-        "description": "Specifies whether to automatically generate `*.spec.ts` files in the Cypress e2e app generated by the `cypress-configure` generator."
-      },
-      "cypressProject": {
-        "type": "string",
-        "description": "The Cypress project to generate the stories under. This is inferred from `name` by default."
-      },
```

Comment:
Bring these back in the `schema.json`. Add a deprecation message in the schema. Throw an error in the generator saying that this was deprecated and has been removed.